smart-cart/app/controllers/lists_controller.rb

```diff
@@ -41,7 +41,7 @@ def create
     #   end
     # end
 
-    List.create(list_id: $session_id, item: params[:item], quantity: 1)
+    @list = List.create(list_id: $session_id, item: params[:item], quantity: 1)
 
```

Comment:
maybe we could add like:

```suggestion
if (!List.find_by(list_id: $session_id, item: params[:item]).empty?)
    throw some sort of flash error or whatever it is
end
```